CONTRIBUTING.md

```diff
@@ -22,8 +22,14 @@ ask you to make any refinements needed or merge it and make them ourselves.
 Things that should go into your PR description:
  * A changelog entry in the `Notes` section (see below)
  * References to any bugs fixed by the change (in GitHub's `Fixes` notation)
- * Notes for the reviewer that might help them to understand why the change is
-   necessary or how they might better review it.
+ * Describe the why and what is changing in the PR description so it's easy for
+   onlookers and reviewers to onboard and context switch.
+ * Include both **before** and **after** screenshots to easily compare and discuss
+   what's changing.
```

Comment:
screenshots probably aren't needed at this level

Reply:
This contributing guide linked from the `matrix-react-sdk` project where it would be highly useful, https://github.com/matrix-org/matrix-react-sdk#developer-guide

Reply:
One day we should stop inheriting our docs from other projects, but today is not that day...